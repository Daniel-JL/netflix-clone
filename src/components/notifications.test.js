import React from 'react';
import { fireEvent, render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import { Notifications } from './notifications';

describe('Notifications dropdown', () => {
  it('should render correctly', () => {
    const wrapper = renderer.create(<Notifications />).toJSON();

    expect(wrapper).toMatchSnapshot();
  });

  it('should activate on mouseover and deactivate on de-mouseover', () => {

  });

  it('should increase slightly the icon size on first click', () => {

  });

  it('should deactivate on second mouse click and the toggle on subsequent clicks', () => {

  });

  it('should open episode and info box if dropdown item is clicked', () => {

  });

  it('should highlight item on mouseover', () => {

  });
});
