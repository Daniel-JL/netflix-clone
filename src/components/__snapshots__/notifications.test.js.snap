// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notifications dropdown should render correctly 1`] = `<div />`;
